tests/tests.rs

```diff
@@ -78,6 +78,11 @@ fn run_cbindgen(
         output,
         str::from_utf8(&cbindgen_output.stderr).unwrap_or_default()
     );
+    if (language, style, cpp_compat) == (Language::C, Some(Style::Type), true) {
+        std::io::stderr()
+            .write_all(&cbindgen_output.stderr)
+            .unwrap();
+    }
```

Comment:
This looks somewhat drive-by? Is this intentionally left here?

Reply:
Oops, sorry